### DISCUSSION TITLE
Linkerd to route traffic between services

### DISCUSSION THREAD

**Discussion by aankam** [linkerd/linkerd2#4444] [General] [answered]
With the multi-cluster discovery, does Linkerd route requests to service pods with zone/region awareness i.e., a service request originated from one zone would be routed to the service from the same zone/region before being routed to the service on a different zone/region/cluster? 

I appreciate your help here. Thanks!

**Comment by grampelberg** [comment, marked as answer]
You can do this manually today with `TrafficSplit`, pick the remote service you'd like to send requests to and set the weight to 100%.

For something a little bit more thorough, check out #4325 for the tracking issue and linkerd/rfc#23 for a look at what we're thinking there.
